udf/bits28_range.sql

```diff
@@ -0,0 +1,40 @@
+/*
+
+Return an INT64 representing a range of bits from a source bit pattern.
+
+See detailed docs for the bits28 suite of functions:
+https://docs.telemetry.mozilla.org/cookbooks/normandy_events.html#udf-reference
```

Comment:
This function might be especially confusing to people. Offhand it's not clear: is this "just the activity, starting at start_offset, and extending n_bits", or is it "what bits$n_bits would be, start at start_offset"? If that isn't clear, it's the difference between not shifting the results to the end, and shifting them (as we actually do below).

Reply:
For this function, it might be nice if we can link to documentation about it explicitly, not just a general reference to the bits28 cookbook.